site/react/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from 'react'
 
+// eslint-disable-next-line import/extensions, node/file-extension-in-import
 import { listFrameworks } from '../../dist'
```

Comment:
Would `../../dist/index.js` work instead?

Reply:
I think we have some conflicting rules (this one conflicts with `https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v23.0.0/docs/rules/import-index.md`)

Reply:
Oh you're right.
We probably should update the linting configuration to be consistent there, either:
  - Enforce requiring `index.js` explicitely
  - Disallow requiring `index.js` explicitely

I would personally favor enforce explicit `index.js` require, because the implicit `index.js` behavior is CommonJS legacy, which does not work well when using ES imports. I remembered Ryan Dahl listed this behavior as one of his top 10 regrets with Node.js (something along the line of "I thought it was cute" :) )

However, either way works as long as we're consistent and enforcing it. What are your thoughts on this?

Reply:
Done in https://github.com/netlify/eslint-config-node/pulls